pkg/querier/querier.go

```diff
@@ -55,8 +55,13 @@ type Config struct {
 	EngineConfig engine.Config `yaml:",inline"`
 }
 
+const (
+	queryIngestersWithinFlag = "querier.query-ingesters-within"
+	queryStoreAfterFlag      = "querier.query-store-after"
+)
+
 var (
-	errBadLookbackConfigs                             = errors.New("bad settings, query_store_after >= query_ingesters_within which can result in queries not being sent")
+	errBadLookbackConfigs                             = fmt.Errorf("the -%s setting must be greater than -%s otherwise queries may return partial results", queryIngestersWithinFlag, queryStoreAfterFlag)
```

Comment:
```suggestion
	errBadLookbackConfigs                             = fmt.Errorf("the -%s setting must be greater than -%s otherwise queries might return partial results", queryIngestersWithinFlag, queryStoreAfterFlag)
```